lib/cadet/assessments/assessments.ex

```diff
@@ -12,44 +12,31 @@ defmodule Cadet.Assessments do
   alias Cadet.Assessments.Assessment
   alias Cadet.Assessments.Question
 
-  def all_assessments() do
-    Repo.all(Assessment)
-  end
-
   def all_assessments(assessment_type) do
-    Repo.all(from(a in Assessment, where: a.type == ^assessment_type))
+    Assessment
+    |> where([a], a.type == ^assessment_type)
+    |> Repo.all()
   end
 
   def all_open_assessments(assessment_type) do
-    now = Timex.now()
-
-    assessment_with_type = Repo.all(from(a in Assessment, where: a.type == ^assessment_type))
-    # TODO: Refactor to be done on SQL instead of in-memory
-    Enum.filter(assessment_with_type, &(&1.is_published and Timex.before?(&1.open_at, now)))
+    Assessment
+    |> where([a], a.is_published)
+    |> where([a], a.open_at <= from_now(1, "second"))
```

Comment:
why not 0 second lol

Reply:
Bump ^

Reply:
oh just wanted to give a little buffer, although I guess it makes no difference either way